SnapWar/Components/Common/SWFullscreenImageVIew.swift

```diff
@@ -0,0 +1,39 @@
+import UIKit
+
+class SWFullscreenImageView: UIView {
+    var imageView: UIImageView
+    
+    required init(named: String) {
+        guard let image = UIImage(named: named) else {
+            fatalError("Image name '\(named)' not found")
+        }
+        self.imageView = UIImageView(image: image)
+        
+        super.init(frame: .zero)
+        
+        setup()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func setup() {
+        backgroundColor = ColorType.black
+        fitToView()
+        addSubview(imageView)
```

Comment:
could we try adding two images to the screen? i can help build out the higher level abstraction later to feed in the images

Reply:
actually, maybe this might get complicated because of landscape, should we just disable the orientation support? like just use portrait.

Reply:
this component is meant to display only a single image and takes up the entire screen

Reply:
ahh true ok